src/utilities/formatDate.ts

```diff
@@ -15,9 +15,9 @@ const monthNames: { [index: string]: string } = {
   '12': i18n.t(langTokens.date.decemberGenitiveCase),
 };
 
-export const formatDate = (dateString = '', time = false): string => {
-  if (dateString.length) {
-    const fullDate = new Date(dateString);
+export const formatDate = (dateValue = '', time = false): string => {
```

Comment:
but what is the type of dateValue? is not it only `String` without `number`?

Reply:
added update